wiki/osu!_wiki/Contribution_guide/GitHub_web-based_editor/de.md

```diff
@@ -0,0 +1,71 @@
+
+::: Infobox
+![](img/command-palette.gif "Beispielhafte Benutzung der Befehlspalette")
+:::
+
+Zwar sind alle Menüs in der Benutzeroberfläche des Editors verfügbar, aber die vorgesehene Art der Navigation ist über die **Befehlspalette**:
+
+- Drücke `F1` und gebe den Namen der zu öffnenden Einstellung oder der auszuführenden Aktion ein. Falls nichts angezeigt wird, probiere das Hamburger-Menü (`≡`) in der linken oberen Ecke des Bildschirms aus.
+- Um eine Datei zu öffnen, drücke `Strg` + `P` und gebe ihren Namen ein.
+
+## Mit Branches arbeiten
+
+Erstelle einen neuen Branch, nachdem du [Bewährte Methoden § Änderungen vornehmen](/wiki/osu!_wiki/Contribution_guide/Best_practices#änderungen-vornehmen) gelesen hast, um deine Änderungen abzuspeichern.
+
+1. Klicke auf den Namen des aktuellen Branches in der linken unteren Ecke oder drücke `F1` und tippe `branch` ein.
+
+  - Wähle `Create new branch...` aus, trage den Namen des Branches ein und drücke `Enter`.
+  - Damit du nicht vergisst, worum es in deiner Arbeit in etwa geht, wähle einen beschreibenden Namen. Beispielsweise kannst du für eine deutsche Übersetzung der Seite [Beatmap-Diskussion](/wiki/Beatmap_discussion) `de-modding-v2` benutzen.
+
+2. Wähle den Namen des entsprechenden Branches im zuvor genannten Dropdown-Menü, um zu deinen Änderungen zurückzukehren.
+
+## Bearbeiten
+
+### Bereits vorhandene Dateien
+
+1. Drücke `Strg` + `P`, gebe den Namen einer Datei ein, die du öffnen möchtest, und tippe dann auf `Enter`. Lose Übereinstimmung wird unterstützt: Zum Beispiel wird durch die Eingabe von `nominators veto de` die Datei `wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto/de.md` aufgerufen.
+2. Bearbeite die Datei nach Bedarf. Nicht in die Versionshistorie aufgenommene Änderungen werden **in deinem Browser gespeichert** und du kannst zu ihnen zurückkehren, nachdem du `github.dev` verlassen hast.
+3. [Erstelle einen Commit](#änderungen-in-die-versionshistorie-aufnehmen), sobald du mit dem/den Artikel(n) zufrieden bist.
+
+### Neue Artikel oder Übersetzungen
+
+Artikel werden innerhalb von Ordnern mit dem originalen Text (`en.md`) und seinen Übersetzungen gespeichert, die verschiedene sprachbasierte Dateinamen nutzen.
+
+- Um eine neue **Übersetzung** für einen existierenden Artikel hinzuzufügen, klicke mit der rechten Maustaste in den jeweiligen Ordner und lege eine neue Datei mit der Endung `.md` und [einem der unterstützten Dateinamen](/wiki/Article_styling_criteria/Formatting#locales) an.
+- Mache das Folgende für das Erstellen eines neuen **Artikels**:
+  - Erstelle einen Ordner in einer angemessenen Kategorie, die der [Namenskonvention](/wiki/Article_styling_criteria/Formatting#ordner--und-dateistrukturen) folgt. Wenn der Artikel nirgendwo hin gehört, dann kreiere einen Ordner im Verzeichnis `/wiki/`.
+  - Füge die Datei `de.md` mit dem Text des Artikels zum neuen Ordner hinzu.
```

Comment:
```suggestion
  - Füge die Datei `en.md` mit dem Text des Artikels zum neuen Ordner hinzu.
```

From my understanding, one would never write *new* articles in german first.

Reply:
[`Guides/Searching_and_Downloading_Beatmaps`](https://osu.ppy.sh/wiki/en/Guides/Searching_and_Downloading_Beatmaps) still waiting for a translation to this day lol